External/Atlases/Desikan_Killiany_License.txt

```diff
@@ -0,0 +1,17 @@
+================================================ LICENSE ================================================
```

Comment:
We tried to use the exact same name for the license txt files and only to add the postfix _License so far. Check out the other examples. Easy fix 👍